manifests/profile/postgres.pp

```diff
@@ -0,0 +1,23 @@
+class puppet_metrics_dashboard::profile::postgres (
+  $pg_password
+){
+
+  pe_postgresql::server::pg_hba_rule{ 'allow telegraf user':
```

Comment:
pe_postgresql breaks using this on open source puppet. I am not sure if we tested it with open source, but maybe we can add the puppetlabs-postgresql module instead? I am not sure if that module provides the same capabilities.